build/pipelines/templates/build-console-steps.yml

```diff
@@ -147,7 +147,6 @@ steps:
 
 - task: PublishBuildArtifacts@1
   displayName: 'Publish Artifact: binlog'
-  condition: failed()

```

Comment:
i think we need specifically something like `condition=always()` otherwise the default is success only

Reply:
oh is that true? ok.

Reply:
may require validation w their yaml spec

Reply:
validated by eyeballs through https://docs.microsoft.com/en-us/azure/devops/pipelines/process/conditions?view=azure-devops&tabs=yaml